oauth2/src/main/java/com/okta/spring/boot/oauth/OktaOAuth2Configurer.java

```diff
@@ -92,16 +116,15 @@ private void configureLogin(HttpSecurity http, OktaOAuth2Properties oktaOAuth2Pr
         }
     }
 
-    private void configureResourceServer(HttpSecurity http, OktaOAuth2Properties oktaOAuth2Properties) throws Exception {
+    private void configureResourceServerForOpaqueTokenValidation(HttpSecurity http) throws Exception {
```

Comment:
Where is this called for non-root issuers?

Reply:
This method will not be called for `non-root` issuer case. Because, for a `non-root` issuer case, the flow could take either `jwt` or `Opaque` token flow depending on what the user has provisioned/configured.

Reply:
it will be invoked after we inspect the jwtconfigurer and determine that jwt is not configured.